File: kotlin/raindrops/src/main/kotlin/Raindrops.kt
object Raindrops {
    fun convert(n: Int) = buildString {
      if (n % 3 == 0) append("Pling")
      if (n % 5 == 0) append("Plang")
      if (n % 7 == 0) append("Plong")
    }.ifEmpty { return n.toString() }
}

File: lua/raindrops/raindrops.lua
return function(n)
  local drops = (n % 3 == 0 and 'Pling' or '') ..
      (n % 5 == 0 and 'Plang' or '') .. (n % 7 == 0 and 'Plong' or '')
  return #drops > 0 and drops or tostring(n)
end
